modules/dnn/include/opencv2/dnn/tokenizer.hpp

```diff
@@ -4,18 +4,21 @@
 #include <string>
 #include <vector>
 
-#include "encoding.hpp"
+// #include "../../src/tokenizer/encoding.hpp"
+#include "../../../src/tokenizer/encoding.hpp"
```

Comment:
The header is a part of interface and binary distribution. Such includes are not allowed.